core/src/context/builder.rs

```diff
@@ -74,14 +72,6 @@ pub mod intrinsic {
         Promise JS_AddIntrinsicPromise,
         /// Add BigInt support
         BigInt JS_AddIntrinsicBigInt,
-        /// Add BigFloat support
-        BigFloat JS_AddIntrinsicBigFloat,
-        /// Add BigDecimal support
-        BigDecimal JS_AddIntrinsicBigDecimal,
-        /// Add operator overloading support
-        Operators JS_AddIntrinsicOperators,
-        /// Enable bignum extension
-        BignumExt JS_EnableBignumExt (1),
```

Comment:
You removed the intrinsics which are no longer in quickjs-ng but not added the new ones. They should be added.